tests/main/ubuntu-core-gadget-config-defaults/gadget-ssh-oneline.yaml

```diff
@@ -0,0 +1,7 @@
+defaults:
+   TEST_SNAP_ID:
+     a: A
+     b: B
+   system:
+     service:
```

Comment:
as we discussed  this doesn't work but

     service.ssh.disable: true

should and we need to keep it working.

Reply:
it is includede in the last commit